gojira.sh

```diff
@@ -819,7 +819,12 @@ main() {
     p_compose down -v
     ;;
   shell)
-    p_compose exec --index $GOJIRA_CLUSTER_INDEX $GOJIRA_TARGET gosh -l -i
+    local opt_envvars
+    # if requested, we set environment variables
+    for binding in "${_GOJIRA_CLI_ENVS[@]}"; do
+      opt_envvars+="  --env ${binding}"
+    done
+    p_compose exec $opt_envvars --index $GOJIRA_CLUSTER_INDEX $GOJIRA_TARGET gosh -l -i
```

Comment:
Looking at it, I think we can move all of this functionality to the common `run_command` with a bit of poking. Once that command supports envs, then any `run|shell` will also support envs

Reply:
This would be here `run_command $GOJIRA_TARGET $GOJIRA_CLUSTER_INDEX gosh -l -i`

Reply:
@eskerda thank you for the suggestions! Taking a look at refactoring with `run_command`.

Reply:
@eskerda refactored to utilizing `run_command` in https://github.com/Kong/gojira/pull/10/commits/7072eb92c330db09aacf286413b00311f16d3fb9